src/samples/Uno.Themes.Samples/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj

```diff
@@ -1,205 +1,203 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build"
-	xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	<Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
-	<ItemGroup>
-		<PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-			<!--
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
might want to avoid changing the whole file...

`git checkout master -- src/samples/Uno.Themes.Samples/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj`
manually edit this line in vs code: `<Version>2.6.0-prerelease.210113001</Version>`
(f1 > save without formating) or (<kbd>alt</kbd>+<kbd>shift</kbd>+<kbd>s</kbd>)